### DISCUSSION TITLE
Is there a showcases page?

### DISCUSSION THREAD

**Discussion by biapar** [keystonejs/keystone#6364] [Event Q&A Q3 2021]
Is there a showcases page?

**Comment by raveling** [comment]
@biapar we don't currently have a showcase page, but we do intend to build one in the future. Because we've been actively developing Keystone 6, there aren't a lot of v6 projects out in the wild yet. That should change as soon as v^ reaches General Availability status.

If you have something to share with the community, be sure to post it to the [community showcase channel](https://keystonejs.slack.com/archives/C01SKLJJN85) in Slack. That's a great place to promote your work, and is a handy reference for when we do publish the showcase. 🙂

**Comment by raveling** [comment]
@biapar here's the video response from the recent community Q&A event:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=1316s

**Reply by biapar** [reply]
The video is not available.
